site/content/en/docs/admission-check-controllers/provisioning.md

```diff
@@ -14,6 +14,7 @@ The Provisioning Admission Check Controller is supported on [Kubernetes cluster-
 
```

Comment:
It looks like we are missing a section that explains how to configure the ProvisioningRequest into a ClusterQueue.